src/find/matchers/perm.rs

```diff
@@ -271,8 +280,8 @@ impl PermMatcher {
     }
 
     #[cfg(not(unix))]
-    pub fn new(pattern: &str) -> Result<PermMatcher, Box<Error>> {
-        Err(From("Permission matching is not available on this platform"))
+    pub fn new(_dummy_pattern: &str) -> Result<PermMatcher, Box<Error>> {
```

Comment:
In a previous review, Raph preferred more explicit names rather than a single underscore...